src/atoms/typography/Icon/glyphs/index.js

```diff
@@ -43,6 +44,7 @@ export {
   CloudDatabase,
   Delete,
   Diagram,
+  DismissData,
```

Comment:
@ej9x why `dismiss`? may be `NoData`?

Reply:
I got this icon from the icons pack. It named there like DismissData.